File: stratego-backend/src/main/java/com/pdrosoft/stratego/dto/GameStateDTO.java
package com.pdrosoft.stratego.dto;

import java.io.Serializable;
import java.util.List;

import com.pdrosoft.matchmaking.dto.PlayerDTO;
import com.pdrosoft.stratego.enums.GamePhase;

import lombok.Builder;
import lombok.Data;

@Data
@Builder
public class GameStateDTO implements Serializable {

	private static final long serialVersionUID = 3924386161467219822L;

	private PlayerDTO currentPlayer;
	private Long gameId;

	private StrategoMovementDTO movement;
	private GamePhase phase;

	private List<List<BoardTileDTO>> board;
	
	private boolean isMyTurn;
}

File: stratego-frontend/Assets/Scripts/ui/dto/GameStateDTO.cs
using System;
using System.Collections.Generic;

[Serializable]
public class GameStateDTO
{
	public PlayerDTO currentPlayer;
	public int hostPlayerId;
	public int guestPlayerId;
	public int gameId;

	public StrategoMovementDTO movement;
	public GamePhase phase;

	public List<List<BoardTileDTO>> board;

	public bool isMyTurn;

}
